src/routes/api/notify/+server.ts

```
@@ -9,9 +9,29 @@ import { isFCMConfigured, sendFCMNotification } from '$lib/modules/server/fcm/fc
 import { json } from '@sveltejs/kit';
+import { existsSync, readFileSync } from 'fs';
+import { homedir } from 'os';
+import { join } from 'path';
 
 import type { RequestHandler } from './$types';
 
+// Reads tokens persisted by /api/device-token — populated automatically when
+// the Android or iOS app registers on first launch.
+function readPersistedDeviceToken(platform: 'android' | 'ios'): string | undefined {
+  const tokensFile = join(homedir(), '.shooter', 'device-tokens.json');
+  if (!existsSync(tokensFile)) return undefined;
```

ESLint

[error] Expected { after 'if' condition.

(curly)